### PR TITLE
Translate_KR:Quote Strings ~

### PR DESCRIPTION
**Pull request by nuguni**
Signed-off-by: nuguni <nuguni@kakao.com>

### CODE REVIEW

File: content/ko/docs/howto/charts_tips_and_tricks.md [hunk 1]
```diff
@@ -41,22 +41,23 @@ value: {{ required "A valid .Values.who entry required!" .Values.who }}
...
+쌍따옴표로 값을 묶는 것이 안전합니다.:
 
 ```yaml
 name: {{ .Values.MyName | quote }}
 ```
 
-But when working with integers _do not quote the values._ That can, in many
+하지만 integer는 많은 경우에 쿠버네티스에서 파싱에러가 발생할 수 있으니 
+_쌍따옴표를 사용하지 마세요._
+when working with integers _do not quote the values._ That can, in many
```
Comment (angellk) [inline content/ko/docs/howto/charts_tips_and_tricks.md:53]: Hi @nuguni - it looks like this line still needs to be removed.

**Review by angellk — COMMENTED** [review]
Hi @nuguni - thanks for your contribution! It looks like your translation hasn't been finished yet on this file. Once it's complete it can be approved!

File: content/ko/docs/howto/charts_tips_and_tricks.md (hunk 1 above)
Comment (nuguni) [reply content/ko/docs/howto/charts_tips_and_tricks.md]: Hi @angellk  I removed it. Thank you.

File: content/ko/docs/howto/charts_tips_and_tricks.md [hunk 2]
```diff
@@ -66,7 +67,7 @@ env:
     value: "1234"
 ```
 
-##'include' 함수 사용하기
+## 'include' 함수 사용하기
 
 Go provides a way of including one template in another using a built-in
```
[the remark this answers is not in the document]
Comment (nuguni) [reply content/ko/docs/howto/charts_tips_and_tricks.md:70]: Hi @angellk  I think i didn't fully understand that section yet. I just wanna correct a typo. Sorry.

**Comment by angellk** [comment]
great - thanks @nuguni ! it looks like your commit just needs to be signed and then this can be approved. look here for info: https://github.com/helm/helm-www/pull/1160/checks?check_run_id=3209321974

**Comment by nuguni** [comment]
Thank you @angellk. Please review again.

**Review by angellk — APPROVED** [review]
/lgtm

**Pull request merged**